firmware/baseband-test/dsp_fft_test.cpp

```diff
@@ -0,0 +1,40 @@
+#include "dsp_fft.hpp"
+#include "doctest.h"
+
+TEST_CASE("ifft") {
```

Comment:
Can we please have a more descriptive name for this test case please. E.g "ifft success with valid input". We want something g that is easily human readable that tells us exactly what we are testing. This looks like a positive test case, so we also want to talk about it being on success as we in the future will also have negative test cases which we will describe as on failure

Reply:
good point. I've gone through all changes and improved a little more.